### ISSUE TITLE
Migrate to Carbon2?

### ISSUE THREAD

**Issue by tdutrion** [opencfp/opencfp#1233]
# Steps required to reproduce the problem

1. clone the project
2. run `composer run setup-env`

# Expected Result

* installation succeed

# Actual Result

* a line in the composer install states: `Carbon 1 is deprecated, see how to migrate to Carbon 2.`

```
Carbon 1 is deprecated, see how to migrate to Carbon 2.
https://carbon.nesbot.com/docs/#api-carbon-2
    You can run './vendor/bin/upgrade-carbon' to get help in updating carbon and other frameworks and libraries that depend on it.
```
Not sure how much work is involved, I have never used this solution.

**Comment by chartjes** [comment]
@tdutrion Carbon 1 is a dependency of illuminate/support, which is in turn a dependency of wouterj/eloquent-bundle. Until such time the use of Eloquent is removed, this warning will be displayed. I will add notes to the README file that this is an expected warning message

**Comment by drbyte** [comment]
> @tdutrion Carbon 1 is a dependency of illuminate/support, which is in turn a dependency of wouterj/eloquent-bundle. Until such time the use of Eloquent is removed, this warning will be displayed.

True, when still using illuminate/support versions less than 5.8.
It appears wouterj/eloquent-bundle is readying an update (1.1) to support newer versions: https://github.com/wouterj/WouterJEloquentBundle/releases

**Comment by chartjes** [comment]
@drbyte That is good to know! I have medium-turn plans to get rid of the dependency on Eloquent, which is only there because I am using cartalyst/sentinel for auth. I believe the process that will have to be done is:

* add Doctrine dependencies to project
* create all the Doctrine entities needed
* refactor existing code to use Doctrine instead of Eloquent
* remove cartalyst/sentinel
* add in symfony/security
* refactor code to use Guard for auth
* profit! (not sure about this step)

**Comment by drbyte** [comment]
> get rid of the dependency on Eloquent, which is only there because I am using cartalyst/sentinel for auth

Sentinel was no doubt a suitable and ideal design choice at the time it was implemented.
But, as you mention, now having something more specific to the chosen framework is more ideal. 
Sounds like the right direction to me.

> * profit! (not sure about this step)

That's always the hope, right!? Especially after doing all the work of the prior steps :P
